Branch-SDK/src/io/branch/referral/SystemObserver.java

```diff
@@ -324,7 +324,8 @@ public boolean getBluetoothPresent() {
         try {
             BluetoothAdapter bluetoothAdapter = BluetoothAdapter.getDefaultAdapter();
             if (bluetoothAdapter != null) {
-                return bluetoothAdapter.isEnabled();
+				//noinspection ResourceType
```

Comment:
"Invite library required Bluetooth permission". Really?

You don't see it when a Branch-SDK user uses the #getBluetoothPresent() function through its callers, or directly. Doesn't mean that it shouldn't be annotated for internal purposes.

That said, imo this entire class should not be public.